### ISSUE TITLE
Migrate  to vike

### ISSUE THREAD

**Issue by wyrd-code** [timnghg/elysia-vite-plugin-ssr#1]
Hi!

As you  can see on the [vite-plugin-ssr](https://vite-plugin-ssr.com/) web, the plugin has been renamed to vike.
Do you have plans to maintain this repo and migrate to that package? Ty

```
vite-plugin-ssr has been renamed [Vike](https://vike.dev/), see [migration guide](https://vite-plugin-ssr.com/vike).
```

**Comment by timnghg** [comment]
Yes, sure, expected next week. I will inform you when it is ready.

**Comment by SelfhostedPro** [comment]
Hey, any update on this?

**Comment by rtritto** [comment]
@timnghg any news?

**Comment by SelfhostedPro** [comment]
> @timnghg any news?

https://github.com/kravetsone/elysia-vite-server May be a good alternative

**Comment by rtritto** [comment]
We should use [vike-node](https://github.com/vikejs/vike-node) (officially maintained by **vike**) that supports _Elysia_

**Comment by SelfhostedPro** [comment]
Oh nice! Good find!
